config/routes.rb

```diff
@@ -283,24 +286,32 @@
     post '/sign_up/completed' => 'sign_up/completions#update'
     get '/user_authorization_confirmation' => 'users/authorization_confirmation#new'
     post '/user_authorization_confirmation' => 'users/authorization_confirmation#create'
-    match '/user_authorization_confirmation/reset' => 'users/authorization_confirmation#destroy', as: :reset_user_authorization, via: %i[put delete]
+    match '/user_authorization_confirmation/reset' => 'users/authorization_confirmation#destroy',
+          as: :reset_user_authorization, via: %i[put delete]
     get '/sign_up/cancel/' => 'sign_up/cancellations#new', as: :sign_up_cancel
     delete '/sign_up/cancel' => 'sign_up/cancellations#destroy', as: :sign_up_destroy
 
     get '/redirect/return_to_sp/cancel' => 'redirect/return_to_sp#cancel', as: :return_to_sp_cancel
-    get '/redirect/return_to_sp/failure_to_proof' => 'redirect/return_to_sp#failure_to_proof', as: :return_to_sp_failure_to_proof
+    get '/redirect/return_to_sp/failure_to_proof' => 'redirect/return_to_sp#failure_to_proof',
+        as: :return_to_sp_failure_to_proof
     get '/redirect/help_center' => 'redirect/help_center#show', as: :help_center_redirect
     get '/redirect/contact/' => 'redirect/contact#show', as: :contact_redirect
 
     match '/sign_out' => 'sign_out#destroy', via: %i[get post delete]
 
     get '/restricted' => 'banned_user#show', as: :banned_user
 
+    get '/errors/idv_unavilable' => 'idv/unavailable#show', as: :idv_unavailable
+
     scope '/verify', as: 'idv' do
       get '/' => 'idv#index'
       get '/activated' => 'idv#activated'
     end
     scope '/verify', module: 'idv', as: 'idv' do
+      if !FeatureManagement.idv_available?
+        # IdV has been disabled.
+        match '/*path' => 'unavailable#show_without_redirect', via: %i[get post]
```

Comment:
Would pointing this directly to `#show` make any difference? It'd also help avoid introducing a non-conventional action method.

```suggestion
        match '/*path' => 'unavailable#show', via: %i[get post]
```

Reply:
Yes, I think this is a good suggestion. I was originally worried about potential redirect loops here, but I'm going to modify this to just use `show`.